Gemfile

```diff
@@ -1,13 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'RedCloth', '4.2.9'
-gem 'jekyll', '3.0.0'
+gem 'RedCloth', '4.3.2'
+gem 'jekyll', '3.6.2'
 gem 'net-sftp', '2.1.2'
-gem 'asciidoctor', '1.5.3'
-gem 'kramdown', '1.6.0'
-gem 'rouge', '1.8.0'
+gem 'asciidoctor', '1.5.6.1'
+gem 'kramdown', '1.16.2'
+gem 'rouge', '2.2.1'
 gem 'less', '2.6.0'
-gem 'therubyracer', '0.12.1'
+gem 'therubyracer', '0.12.3'
 gem 'curb'
 gem 'diskcached'
 gem 'html-proofer'
+
```

Comment:
@pmacko1 could you please add versions to all Gems?
we were running into some problems because of this: https://github.com/aerogear/aerogear.org/commit/76441ceaf74c78e942b69ae8aba84cd56c1555f3

I am fine with the versions you added there though. I was just trying to fix the issue quickly.